vm-virtio/src/mem.rs

```diff
@@ -695,7 +695,10 @@ impl MemEpollHandler {
                             if let Err(e) = &r {
                                 error!("{:?}", e);
                             }
-                            self.resize.send(r);
+                            if let Err(e) = self.resize.send(r) {
+                                error!("Resize send got {:?}", e);
```

Comment:
This message could be improved. 
`error!("Sending "resize" generated error: {:?}");`

Reply:
In fact this might be worse than the old panicking behaviour as the error gets consumed an never propagated upwards.

The error handling in this whole function is wrong as all the errors created above are never returned as this function always returns `Ok()`

Reply:
Oh yes I missed that, we need the error to be propagated upwards if we want to follow the same behavior.

Reply:
Post a new version that return reize error in MemEpollHandler.run.